pkg/pods/manifest.go

```diff
@@ -134,7 +240,10 @@ func (manifest *Manifest) Write(out io.Writer) error {
 // OriginalBytes() always returns the bytes from the original manifest file
 // that was passed to ManifestFromBytes(). Any mutations to the struct will not
 // appear in the returned bytes. If mutations are desired, use Marshal()
-func (manifest *Manifest) OriginalBytes() ([]byte, error) {
+func (manifest *manifest) OriginalBytes() ([]byte, error) {
```

Comment:
Do we need a `Bytes()` vs `OriginalBytes()` anymore?  Seems like we can remove `OriginalBytes` and make `Bytes` return what `OriginalBytes` returned.